.github/workflows/test.yml

```diff
@@ -8,6 +8,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
+        datasette-version: ["<=1.0a0", ">=1.0a0"]
```

Comment:
@simonw, I am perhaps naive (or inattentive), but these are two inclusive version specifiers. Does it not run the risk of testing against 1.0a0 twice?